sos/report/plugins/ovn_central.py

```diff
@@ -192,7 +192,16 @@ def setup(self):
             cmds, foreground=True, container=self.container_name
         )
 
+        # Collect Certificate Validity Dates
+        for path in ['/etc/ovn/ovn-central.crt', '/etc/ovn/cert_host']:
```

Comment:
Are the certs always going to have these locations? i.e. could they be name differently?

also for the RH folks @pmoravec @jcastill would these be same locations for you?